src/aiai_eval/named_entity_recognition.py

```diff
@@ -18,3 +20,16 @@ class NEREvaluation(EvaluationTask):
         evaluation_config (EvaluationConfig):
             The configuration of the evaluation.
     """
+
+    def _preprocess_data(self, dataset: Dataset, framework: str, **kwargs) -> Dataset:
+        """Preprocess a dataset by tokenizing and aligning the labels.
+        Args:
+            dataset (Hugging Face dataset):
+                The dataset to preprocess.
+            kwargs:
+                Extra keyword arguments containing objects used in preprocessing the
+                dataset.
+        Returns:
+            Hugging Face dataset: The preprocessed dataset.
+        """
```

Comment:
Also missing framework in docstring.

```suggestion
        """Preprocess a dataset by tokenizing and aligning the labels.
        
        Args:
            dataset (Hugging Face dataset):
                The dataset to preprocess.
            kwargs:
                Extra keyword arguments containing objects used in preprocessing the
                dataset.
                
        Returns:
            Hugging Face dataset: The preprocessed dataset.
        """
```